fastify.js

```diff
@@ -183,7 +182,7 @@ function fastify (options) {
       allowUnsafeRegex: options.allowUnsafeRegex || defaultInitOptions.allowUnsafeRegex,
       buildPrettyMeta: defaultBuildPrettyMeta,
       querystringParser: options.querystringParser,
-      useSemicolonDelimiter: options.useSemicolonDelimiter || defaultInitOptions.useSemicolonDelimiter
+      useSemicolonDelimiter: typeof options.useSemicolonDelimiter === 'boolean' ? options.useSemicolonDelimiter : defaultInitOptions.useSemicolonDelimiter
```

Comment:
I guess you wanted to use the `??` operator?

```suggestion
      useSemicolonDelimiter: options.useSemicolonDelimiter ?? defaultInitOptions.useSemicolonDelimiter
```

Reply:
Yes thank you should be updated now